features/product/viewing_products/viewing_product_with_proper_iri_identifiers.feature

```diff
@@ -0,0 +1,29 @@
+@viewing_products
+Feature: Viewing a product with proper iri identifiers
+    In order to see products with proper iri identifiers
+    As a Visitor
+    I want to be able to view a single product
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a "Lamborghini Gallardo Model" configurable product
+        And this product has "Red", "Yellow" and "Black" variants
+        And this product has option "Engine Power" with values "500KM", "750KM" and "1000KM"
+        And the store classifies its products as "Super Car"
+        And this product is in "Super Car" taxon at 1st position
+        And the product "Lamborghini Gallardo Model" has a main taxon "Super Car"
+        And this product has an image "lamborghini.jpg" with "main" type
+        And this product has a review titled "Great Car" and rated 5 added by customer "h.p.lovecraft@arkham.com", created 3 days ago
+
+    @api
+    Scenario: Viewing proper iri identifiers for visitor
```

Comment:
Yes but `spec` test only mock and it does not match to this behawior, Testing it with `unit` its possible but it will take x10(or higher) longer than behats. We must think how test it easier and faster and better, because we need testing all entities.

Reply:
I added new approach to test it look in step: https://github.com/Sylius/Sylius/pull/11925/files#diff-3ea3a89418303ef08443a4292dd9cd26b55365f535efadb9762e47d82631d106R21